components/compose/awesomebar/src/main/java/mozilla/components/compose/browser/awesomebar/internal/Suggestion.kt

```diff
@@ -173,5 +173,6 @@ private fun AutocompleteButton(
                 }
             )
             .clickable { onAutoComplete() }
+            .padding(12.dp)
```

Comment:
Wondering if we should've kept the padding below the `.size()` just so we can see the size and padding right next to each other.

I also realized that the entire file uses `width()`/`height()`, but let's just ignore that for the purpose of solving what we set out for this issue.

Reply:
The padding has to be after the clickable modifier, otherwise, the padding would not be clickable.

Reply:
Makes sense

Reply:
I thought about this some more and perhaps got myself confused, but since modifiers are applied from first to last (left to right, order is important), wouldn't the clickable not apply to the padding since it's after?

Reply:
No, it means we first add the click on the whole item and then apply the padding. If we do that in reverse order we would first add the padding and add the clickable to the remaining size (without padding.)

Reply:
Ok that makes sense